### PR TITLE
[UX] Improve the explanation of the Automatic way to implement the form

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
This update introduces a clearer explanation for the embedded form implementation method. Previously, users lacked the information that any modifications made to the form within the platform would synchronize across all embedded instances. 

This enhancement ensures users are fully aware of the dynamic editing capabilities, leading to a more intuitive and efficient content management experience. This is particularly beneficial for users who manage forms across multiple pages, streamlining their workflow and reducing the need for repetitive updates.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create a form
4. Save & close the form
5. Click on Automatic 
6. Read the description
image

### CODE REVIEW

**Review by Esthertests — APPROVED** [review]
LGTM

on prod
image

On PR 
image

File: app/bundles/FormBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -126,10 +126,10 @@ mautic.form.form.header.edit="Edit Form - %name%"
 mautic.form.form.header.manualcopy="Manual Copy"
 mautic.form.form.header.new="New Form"
 mautic.form.form.header.preview="Preview Form"
-mautic.form.form.help.automaticcopy="With this option, Mautic will automatically generate the form where you include this code and thus it will always be the most up-to-date version."
-mautic.form.form.help.automaticcopy.js="Via Javascript (recommended)"
-mautic.form.form.help.automaticcopy.iframe="Via iframe"
-mautic.form.form.help.automaticcopy.iframe.note="Note: Adjust the width and height attribute so the form fits in."
+mautic.form.form.help.embedded="When you use this option, the entire form can be edited within the platform and these changes will automatically be reflected on any page where it is inserted."
+mautic.form.form.help.embedded.js="Via Javascript (recommended)"
```
Comment (escopecz) [inline app/bundles/FormBundle/Translations/en_US/messages.ini]: Do we need to change the translation keys for translations that aren't changing? This way all the translations in all languages will have to translate this again. Seems like a lot of work for changing a translation key that no one actually sees.

**Review by escopecz — CHANGES_REQUESTED** [review]

File: app/bundles/FormBundle/Translations/en_US/messages.ini (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/FormBundle/Translations/en_US/messages.ini]: Thanks for pointing!

**Review by escopecz — APPROVED** [review]
Thanks!

**Comment by RCheesley** [comment]
Docs created here: https://github.com/mautic/user-documentation/pull/270

**Pull request merged**